src/main/scala/workflow/graph/Estimator.scala

```diff
@@ -0,0 +1,63 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+
+/**
+ * An estimator has a `fitRDD` method which takes an input and emits a [[Transformer]]
```

Comment:
This is another case where the type signature is the same, but the method that needs to be implemented was renamed. This change was done so that calling `est.fit(...)` is now lazy, bringing with it all the lazy/incremental pipeline construction stuff we've been discussing for the past few weeks.